templates/index.html

```diff
@@ -112,7 +112,7 @@
                             {% include 'font_why.html' %}
                         </div>
                         <div class="tab-pane fade svn text-center" id="svn">
-                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}">
+                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}" width="362" height="362">
```

Comment:
width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard